src/Sulu/Bundle/ContentBundle/Resources/public/js/components/smart-content/main.js

```diff
@@ -129,7 +128,6 @@ define([], function() {
             hideCategories: false,
             hideTags: false,
             hideSortBy: false,
-            hidePresentAs: false,
```

Comment:
i have changed the behavior to if no present as options are past it is hidden and vice versa!